<!DOCTYPE HTML>
<!--
	Forty by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<!-- Global site tag (gtag.js) - Google Analytics -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=G-SCYVN71ZQ5"></script>
		<script>
		window.dataLayer = window.dataLayer || [];
		function gtag(){dataLayer.push(arguments);}
		gtag('js', new Date());

		gtag('config', 'G-SCYVN71ZQ5');
		</script>
		<title>Nae-Chyun</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="index.html" class="logo"><strong>Nae-Chyun</strong> <span></span></a>
						<nav>
							<a href="#menu">Menu</a>
						</nav>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<ul class="links">
							<li><a href="index.html">Home</a></li>
							<li><a href="aboutme.html">About Me</a></li>
							<li><a href="research.html">Research</a></li>
							<li><a href="publication.html">Publications</a></li>
							<li><a href="life.html">Life</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main" class="alt">

						<!-- One -->
							<section id="one">
								<div class="inner">
									<header class="major">
										<h1>Research</h1>
									</header>
									<h3>Improving sequence alignment and reducing reference bias</h3>
									<p>
										Alignment is the first step for many genomics applications. 
										Currently, people rely on a linear haploid reference genome such as GRCh38 for this task. 
										However, more than half of the "standard" reference genome is sequenced from one individual. 
										This results in "reference bias" that confounds downstream analyses. </br>

										A solution is to use a reference genome containing population information. 
										Our study, FORGe, presents a ranking strategy for the inclusion of population data in a comprehensive graph genome. 
										We also devlope levioSAM to perform fast lift-over of alignments that are based on a variant-aware reference.
										Further, the reference flow framework uses a small number of additional population-aware linear references to improve alignment. 
									</br>
									<ul>
										<li>Publications: 
											<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02229-3">Chen <i>et al.</i> (2021)</a>, 
											<a href="https://doi.org/10.1093/bioinformatics/btab396">Mun<sup>*</sup>, Chen<sup>*</sup>, Langmead (2021)</a>, 
											<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1595-x">Pritt, Chen, Langmead (2018)</a></li>
										<li>Presentation: <a href="files/2019-Chen-Recomb_seq.pdf">RECOMB-Seq (Washington DC, USA, 2019; poster)</a>, 
											<a href="files/2019-Chen-GenomeInformatics.pdf">Genome Informatics (CSHL, USA, 2019; poster)</a></li>
										<li>Software: 
										<a href="https://github.com/langmead-lab/reference_flow">reference flow</a>, 
										<a href="https://github.com/alshai/leviosam">levioSAM</a>, 
										<a href="https://github.com/langmead-lab/FORGe">FORGe</a>, 
										<a href="https://github.com/BenLangmead/bowtie-majref">bowtie-majref</a></li>
									</ul>
									</p>
									<h3>Accurate variant calling utilizing population data and deep learning</h3>
									<p>
										Variant calling is a genomic data processing method to identify differences between a sample and the reference, which is usually done using sophisticated processing and statistical methods. 
										<a href="https://github.com/google/deepvariant">DeepVariant</a> is a deep-learning-based variant caller developed by Google Health. 
										While interning on the DeepVariant team, I developed a population-aware DeepVariant model to incorporate population information in variant calling.
										This model is more accurate than the default DeepVariant model.
									</br>
									<ul>
										<li>Publications: <a href="https://doi.org/10.1101/2021.01.06.425550">Chen <i>et al.</i> (2021, preprint)</a>
										</li>
										<li>Presentation: Biological Data Science (virtual, 2020; poster)</li>
										<li>Software: <a href="https://github.com/google/deepvariant">DeepVariant</a></li>
									</ul>
									</p>
									<h3>Developing computational methods for biological and medical data</h3>
									<p>
										Many research areas have been adopting modern sequencing techonology for faster and more accurate analysis. 
										I collaborate with researchers at Johns Hopkins University to predict chromatin accessibility using machine learning methods. 
										Also, I work with collaborators at National Taiwan Univeristy to study immunogenomics such as the human leukocyte antigen (HLA) complex and the Adaptive Immune Receptor Repertoire (AIRR).
									</br>
									<ul>
										<li>Publications: <a href=https://doi.org/10.1101/2020.11.27.399857">Lin <i>et al.</i> (2021, preprint)</a></li>
										<li>Presentation: JHU Chromatin Workshop (2019; poster)</li>
									</ul>
									</p>
									<h3>Designing efficient hardware architecture for computational genomics</h3>
									<p>
										With the enormous volume of genomics data, how to process it efficiently has become a challenging topic. 
										One solution is using specialized hardware architecture based on FPGA or ASIC to make computing efficient. 
										I design specialized algorithms to meet hardware requirements (which are usually more limited than that of software) and develop architectures for efficient computing. 
										My research includes short read and long read alignment, multiple sequence alignment, and assembly.
									</br>
									<ul>
										<li>Selected publications: <a href="https://ieeexplore.ieee.org/abstract/document/8445105">Liao <i>et al.</i> (2018)</a>, 
											<a href="files/2018-ISCAS.pdf">Chen <i>et al.</i> (2018)</a>, 
											<a href="https://ieeexplore.ieee.org/abstract/document/7348380">Chen <i>et al.</i> (2015)</a></li>
									<!-- </br> -->
										<li>Presentations: APCCAS (Jeju, Korea, 2016; oral), BioCAS (Atlanta, USA, 2015; poster)</li>
									</ul>
									</p>
								</div>
							</section>

					</div>

				<!-- Footer -->
				<footer id="footer">
					<div class="inner">
						<ul class="icons">
							<li><a href="http://twitter.com/naechyun_chen" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="http://github.com/milkschen" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
							<li><a href="https://www.linkedin.com/in/ncchen/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
							<li><a href="https://scholar.google.com/citations?hl=en&user=JymhwIwAAAAJ&view_op=list_works&sortby=pubdate" class="icon brands alt fa-google"><span class="label">Google Scholar</span></a></li>
							<li><span class="icon solid alt fa-envelope"></span> naechyun< at >jhu.edu</li>
						</ul>
						<ul class="copyright">
							<li>&copy; Untitled</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li>
						</ul>
					</div>
				</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>