<!DOCTYPE HTML>
<!--
	Forty by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Nae-Chyun</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="index.html" class="logo"><strong>Nae-Chyun</strong> <span></span></a>
						<nav>
							<a href="#menu">Menu</a>
						</nav>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<ul class="links">
							<li><a href="index.html">Home</a></li>
							<li><a href="aboutme.html">About Me</a></li>
							<li><a href="research.html">Research</a></li>
							<li><a href="publication.html">Publications</a></li>
							<li><a href="life.html">Life</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main" class="alt">

						<!-- One -->
							<section id="one">
								<div class="inner">
									<header class="major">
										<h1>Research</h1>
									</header>
									<h3>Improving sequence alignment and reducing reference bias</h3>
									<p>
										Alignment is the first step for many genomics applications. 
										Currently, people rely on a linear haploid reference genome such as GRCh37 and GRCh38 for this task. 
										However, more than half of the "standard" reference genomes is from one individual, resulting in confounding of downstream results. 
										A solution is replacing the linear reference with a graph data structure that contains genotypes collected from many individuals, aka graph genome. 
										Our study, FORGe, presents a ranking strategy for the inclusion of population data. 
										As an extension of FORGe, my first-author project, reference flow, uses a small number of additional references built using population data to improve the alignment results. 
									</br></br>
										Publications: Chen <i>et al.</i> (2020, preprint), Pritt <i>et al.</i> (2018)
									</br>
										Presentation: RECOMB-Seq (Washington DC, USA, 2019), Genome Informatics (CSHL, USA, 2019)
									</p>
									<h3>Developing computational methods for biological and medical data</h3>
									<p>
										Many research areas have been adopting modern sequencing techonology for faster and more accurate analysis. 
										I collaborate with researchers at Johns Hopkins University to predict chromatin accessibility using machine learning methods. 
										Also, I work with collaborators at National Taiwan Univeristy to study immunogenomics such as the human leukocyte antigen (HLA) complex.
									</br></br>
										Presentation: JHU Chromatin Workshop (2019)
									</p>
									<h3>Designing efficient hardware architecture for computational genomics</h3>
									<p>
										With the enormous volume of genomics data, how to process it efficiently has become a challenging topic. 
										One solution is using specialized hardware architecture based on FPGA or ASIC to make computing fast or power efficient. 
										I designed algorithms to meet hardware requirements and developed architectures for efficient computing. 
										My research includes short read and long read alignment, multiple sequence alignment, and assembly.
									</br></br>
										Publications: Liao <i>et al.</i> (2018), Chen <i>et al.</i> (2018), Lin <i>et al.</i> (2018), Chang <i>et al.</i> (2016), Liu <i>et al.</i> (2016), Chen <i>et al.</i> (2015)
									</br>
										Presentations: APCCAS (Jeju, Korea, 2016), BioCAS (Atlanta, USA, 2015)
									</p>
								</div>
							</section>

					</div>

				<!-- Footer -->
				<footer id="footer">
					<div class="inner">
						<ul class="icons">
							<li><a href="http://twitter.com/naechyun_chen" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="http://github.com/milkschen" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
							<li><a href="https://www.linkedin.com/in/ncchen/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
							<li><a href="https://scholar.google.com/citations?hl=en&user=JymhwIwAAAAJ&view_op=list_works&sortby=pubdate" class="icon brands alt fa-google"><span class="label">Google Scholar</span></a></li>
							<li><span class="icon solid alt fa-envelope"></span> cnaechy1< at >jhu.edu</li>
						</ul>
						<ul class="copyright">
							<li>&copy; Untitled</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li>
						</ul>
					</div>
				</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>